mkosi/__init__.py

```diff
@@ -1902,31 +1903,54 @@ def reenable_kernel_install(args: CommandLineArguments, root: str) -> None:
     make_executable(hook_path)
 
 
+def add_packages(
+    args: CommandLineArguments, packages: Set[str], *names: str, conditional: Optional[str] = None
+) -> None:
+    """Add packages in @names to @packages, if enabled by --base-packages.
+
+    If @conditional is specifed, rpm-specific syntax for boolean
+    dependencies will be used to include @names on if @conditional is.
+    """
+    assert args.base_packages is True or args.base_packages is False or args.base_packages == "conditional"
+
+    if args.base_packages or (conditional or args.base_packages is True):
```

Comment:
This line seems weird to me. If `base_packages` is `True` or `"conditional"`, the first branch will trigger. If it's `False` the `is True` in the other branch can't trigger anymore.

Reply:
Yep, will fix.

Reply:
Thanks for catching this.  I reworked this as:
```python
if args.base_packages is True or (args.base_packages == 'conditional' and conditional):
    ...
```
I think this is better.